src/collaborating/peer-code-review.md

```diff
@@ -40,6 +40,11 @@ In creating a pull request and inviting someone to review your work, the pull re
 Make the reviewer's job easier by giving them small amounts of code to review.
 ```
 
+## Finding a reviewer
+
+On GitHub we have started a [peer-review team](https://github.com/orgs/spectrum-spark/teams/peer-review).
+We encourage you to post on the discussion board, to find likeminded members to review your code. 
```

Comment:
I think we should say "like-minded" here.

Reply:
I agree.